### PR TITLE
[Reusable Component] Layout-Right-Sidebar

### PR DESCRIPTION
**Pull request by FranzDelosSantos**
## Issue Link
https://framgiaph.backlog.com/board/LMS?selectedIssueKey=LMS-305

## Defintion of Done
able to display Right Side Bar Container

## Steps to reproduce
cd client
npm run dev
visit url: http://localhost:3000/demo/layouts/SidebarLayout

## Affected Components / Functionalities / Page
RightSideBar.tsx

## Test Cases
_will update soon..._

## Notes
It has a margin left of 24px

## Screenshots
image
image

### CODE REVIEW

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
✅ Right side bar container is displayed/visible in the given page http://localhost:3000/demo/layouts/RightSideBar
✅No ESLint warnings or errors.

Comments: 0
Test time: 18 mins
Iterations: 0

File: client/src/shared/layouts/Right Side Bar/RightSideBar.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface rightSideBarProps {
+  children: any
+}
+
+const RightSideBar: React.FunctionComponent<rightSideBarProps> = ({
+  children
+}: rightSideBarProps) => {
+  return (
```
Comment (ragg09-sunasterisk) [inline client/src/shared/layouts/Right Side Bar/RightSideBar.tsx:10]: please add the left container also, follow the layout [here](https://www.figma.com/file/6f92xbjEjJfQvwzbCTOWqF/TRAINING-PROGRAM-LMS?node-id=212-16&t=uIAvq0xwKPruFeG7-0)

File: client/src/shared/layouts/Right Side Bar/RightSideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <div className='sidebar fixed top-0 bottom-0 right-20 bg-gray-400 w-80 mr-12 ml-6 '>
```
Comment (ragg09-sunasterisk) [inline client/src/shared/layouts/Right Side Bar/RightSideBar.tsx:11]: use double quote

**Comment by ragg09-sunasterisk** [comment]
Comments: 2
Iterations: 1

**Pull request merged**